content/sensu-go/6.4/operations/deploy-sensu/scale-event-storage.md

```diff
@@ -435,46 +421,38 @@ After you enter your password, Postgres will list database copy progress:
 30318/30318 kB (100%), 1/1 tablespace
 {{< /code >}}
 
-### Step 4: Confirm replication
-
-To confirm your configuration is working properly, first remove configurations that are only for the primary:
+### Confirm replication
 
+1. On the **primary** Postgres host, remove primary-only configurations:
 {{< code shell >}}
 sudo sed -r -i.bak '/^(wal_level|max_wal_senders|wal_keep_segments).*/d' /var/lib/pgsql/data/postgresql.conf
 {{< /code >}}
 
-Start the PostgreSQL service:
-
+2. Start the PostgreSQL service:
 {{< code shell >}}
 sudo systemcl start postgresql
 {{< /code >}}
 
-To verify that the replication is taking place, check the commit log location on the primary and standby hosts.
-
-From primary:
-
+3. Check the commit log location from the primary host:
 {{< code shell >}}
 sudo -u postgres psql -c "select pg_current_xlog_location()"
 {{< /code >}}
 
-Postgres will list the commit log location:
-
+   Postgres will list the primary host commit log location:
 {{< code postgresql >}}
  pg_current_xlog_location 
 --------------------------
  0/3000568
 (1 row)
 {{< /code >}}
 
-From standby:
-
+3. Check the commit log location from the standby host:
 {{< code shell >}}
 sudo -u postgres psql -c "select pg_last_xlog_receive_location()"
 sudo -u postgres psql -c "select pg_last_xlog_replay_location()"
 {{< /code >}}
 
-Postgres will list the commit log location:
-
+   Postgres will list the standby host commit log location:
```

Comment:
I think something might be missing here -- perhaps users need to confirm identical responses for the primary and standby hosts log locations? As written, the instructions don't explain how the steps here result in a confirmation that SR is working.